main/amavisd-new/amavisd.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+
+pidfile="/run/$RC_SVCNAME.pid"
+
+name="amavisd"
+command="/usr/sbin/$name"
+command_args="-c /etc/amavisd.conf foreground"
+command_background="yes"
+
+start_stop_daemon_args="--user $amvuser:$amvgroup"
```

Comment:
`USER` has a bit special meaning, but `user` has not (shell is case-sensitive). Variables `user` and `group` for this are okay and actually preferred (it’s a common name for this). Sorry that I didn’t mention it before.